tests/src/Kernel/fixtures/markup_rendering_patterns/alert.yml

```diff
@@ -7,20 +7,20 @@ alert_dimissible_animated:
         dismissible: true
         animated_dismiss: true
         style: 'danger'
-      heading: "Well done!"
-      message: "A simple alert. check it out!"
-      icons_path: "/assets/icons/bootstrap-icons.svg"
-      icon_name: "alarm"
+      heading: 'Well done!'
+      message: 'A simple alert. check it out!'
+      icons_path: '/assets/icons/bootstrap-icons.svg'
```

Comment:
pls create a follow up ticket that sets this inside the pattern template and doesn't expose it as a parameter

Reply:
Is already here: https://citnet.tech.ec.europa.eu/CITnet/jira/browse/OEL-779 @drishu